mllib/src/main/scala/org/apache/spark/ml/feature/PCA.scala

```diff
@@ -34,6 +34,7 @@ import org.apache.spark.rdd.RDD
 import org.apache.spark.sql._
 import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.{StructField, StructType}
+import org.apache.spark.util.VersionUtils._
```

Comment:
In general I think we prefer importing classes rather than importing members, except where the latter is clearly clearer. I'm not sure it is here. It's minor either way.

Are there any other occurrences across the code? i want to try to solve these issues in one go.